web-client/src/views/AdvancedSearch/DocumentSearchResults.jsx

```diff
@@ -39,8 +39,8 @@ export const DocumentSearchResults = connect(
             <table className="usa-table search-results ustc-table responsive-table">
               <thead>
                 <tr>
-                  <th aria-hidden="true" className="small-column"></th>
-                  <th aria-hidden="true" className="small-column"></th>
+                  <td aria-hidden="true" className="small-column"></td>
```

Comment:
is this related to aria why it needs to become a td?

Reply:
Explanation found here, I just went with it: https://github.com/flexion/ef-cms/issues/8105